extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/conf/ConfigurationService.java

```diff
@@ -85,4 +85,34 @@ public PrivateKey getClearpassKey() throws GuacamoleException {
         return environment.getProperty(CASGuacamoleProperties.CAS_CLEARPASS_KEY);
     }
 
+    /**
+     * Returns the attribute used to determine group memberships
+     * in CAS, or null if not defined.
+     *
+     * @return
+     *     The attribute name
+     *
+     * @throws GuacamoleException
+     *     If guacamole.properties cannot be parsed.
+     */
+    public String getCasGroupAttribute() throws GuacamoleException {
+        return environment.getProperty(CASGuacamoleProperties.CAS_GROUP_ATTRIBUTE);
+    }
+
+    /**
+     * Returns the attribute used to format group DN's
+     * in CAS, or null if not defined.
```

Comment:
This seems a little unclear to me.  I know you described the behavior in the body of the pull request, so I think I get what you're trying to say, but try to think of it from the perspective of someone reading JavaDocs without having ever seen what you wrote on the pull request, and maybe without having read any additional stuff that will be added to the manual.  So, this should be a little more descriptive of what the property is used for.

Reply:
I've taken a crack at a correction to this, please review and let me know if it's clearer.